pkg/dao/suite_test.go

```diff
@@ -45,27 +45,34 @@ type RepositorySuite struct {
 
 var orgIDTest = seeds.RandomOrgId()
 var accountIdTest = seeds.RandomAccountId()
+var timestamp = time.Now()
 
 var repoPublicTest = models.Repository{
 	Base: models.Base{
 		CreatedAt: time.Now(),
 		UpdatedAt: time.Now(),
 	},
-	URL:           "https://www.redhat.com",
-	LastReadTime:  nil,
-	LastReadError: nil,
-	Public:        true,
+	URL:                          "https://www.redhat.com",
+	Public:                       true,
+	LastIntrospectionTime:        &timestamp,
+	LastIntrospectionUpdateTime:  &timestamp,
+	LastIntrospectionSuccessTime: &timestamp,
+	LastIntrospectionError:       nil,
+	Status:                       "Valid",
```

Comment:
I suggest to replace the string by a constant; suggested ones below:

```golang
const (
	StatusValid       = "Valid"       // Repository introspected successfully
	StatusUnavailable = "Unavailable" // Repository introspedted but was unavailable
	StatusInvalid     = "Invalid"     // Repository introspected but is invalid
	StatusPending     = "Pending"     // Repository not introspected yet
)

```

Reply:
good idea! updated to this. I put the constants in `/dao/repositories.go`